libafl_frida/Cargo.toml

```diff
@@ -27,7 +27,7 @@ hashbrown = "0.11"
 libloading = "0.7.0"
 rangemap = "0.1.10"
 frida-gum-sys = { version = "0.3", features = [ "auto-download", "event-sink", "invocation-listener"] }
-frida-gum = { version = "0.5.2", features = [ "auto-download", "backtrace", "event-sink", "invocation-listener"] }
+frida-gum = { version = "0.5.2", features = [ "auto-download", "event-sink", "invocation-listener"] }
```

Comment:
Why remove this?

Reply:
backtracer.rs is not ready for windows.
https://github.com/frida/frida-rust/blob/master/frida-gum/src/backtracer.rs
,plus as far as I see this feature is not used anywhere in libafl_frida.

Reply:
@s1341 does libafl_frida use the backtrace feature for something

Reply:
Seems like it's not needed atm.